src/test/java/uk/gov/pay/directdebit/payments/clients/GoCardlessClientFacadeTest.java

```diff
@@ -100,4 +113,23 @@ public void validateReturnsGoCardlessBankAccountLookupWithNullBankNameAndBacsFal
         assertThat(result.isBacs(), is(false));
     }
 
+    @Test
+    public void createMandateReturnsGoCardlessMandateWithCreditorId() {
+        CreditorId creditorId = CreditorId.of("test-creditor-id-here");
+        uk.gov.pay.directdebit.mandate.model.Mandate mandate =
+                MandateFixture.aMandateFixture().toEntity();
+        GoCardlessCustomer goCardlessCustomer =
+                GoCardlessCustomerFixture.aGoCardlessCustomerFixture().toEntity();
+
+        given(mockMandate.getId()).willReturn("test-mandate-id-here");
+        given(mockMandate.getReference()).willReturn("test-mandate-reference-here");
+        given(mockMandate.getLinks()).willReturn(mockMandateLinks);
+        given(mockMandateLinks.getCreditor()).willReturn(creditorId.toString());
+
+        given(mockGoCardlessClientWrapper.createMandate(any(), any())).willReturn(mockMandate);
```

Comment:
I really discourage using `any()`, there are very few cases where we don't know what the parameters are going to be, especially when we actually set up the mocks ourselves. We have the mandate external Id and also the goCardlessCustomer, let's use them